apps/aecontract/src/aect_contracts.erl

```diff
@@ -142,11 +125,7 @@ serialize(#contract{} = C) ->
       ?CONTRACT_TYPE,
       ?CONTRACT_VSN,
       serialization_template(?CONTRACT_VSN),
-      [ {pubkey, pubkey(C)}
```

Comment:
Could you please propose contextual update of [serialization doc](https://github.com/aeternity/protocol/blob/414051131a29b808bd829525f773e8fc3ae75d2d/serializations.md#contract)?

Reply:
https://github.com/aeternity/protocol/pull/82

Reply:
Thanks